tests/correctness/udot_product.futil

```diff
@@ -4,9 +4,9 @@ import "primitives/bitnum/signed.futil";
 component main() -> () {
```

Comment:
Maybe rename the test file to `unsigned-dot-product.futil`?

Reply:
Done.